loops_lab.js

```diff
@@ -0,0 +1,203 @@
+// <!-- # Loops Lab
+//
+// To complete this lab, do the following:
+//
+// 1. Fork this repo
+// 2. Clone **your** forked repo
+// 3. Create a file called `loops_lab.js` inside your cloned repo
+// 4. Copy the description of the exercise as a comment
+// 5. Underneath the commented description write your code.
+// 6. Ensure your solutions work properly!
+// 7. Open PR against this repo.
+//
+// ---
+//
+// ## Exercises
+//
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1. -->
+//
+//   let num = 23;
+// while (num >= 1) {
+// console.log(num);
+// num -= 1;
+// }
+// console.log("--------------");
+// for (let num = 23; num >= 1; num -= 1) {
+// console.log(num);
+// }
+//
+// 2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "max".
+//    For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+//
+
+//
+// 3. Write a `while` loop and a `for` loop that takes the variable "x" and iterates over all numbers from 0 to "x".
+//    For each iteration of the loop, it will multiply the number of "x" by 9 and log the result (e.g. "2 \* 9 = 18").
+//    _Bonus_ think of another way to solve it.
+// for (let num = 0; num < 36; num += 1) {
+//   console.log(num + ' * 9 =' + num * 9)
+// }
+// console.log('-----------------');
+
+//
+//    <details>
+//    <summary>
+//    Hint
+//    </summary>
+//    Find the final number and increment the loop by 9.
+//    </details>
+//
+// 4. Write code that prints/logs all the numbers that end in a 5 from 1 to 100, **exclusive**.
+// for (x = 0; x < 100; x += 5){
+//   if (x % 2 === 1){
+//     console.log(x);
+//   } else {
+//     console.log("");
+//   }
+// }
+// 5. Without running/executing your code, how many times will the loop below run? Explain why.
+//
+// ```js
+// let i = 5;
+//
+// while (i > 3) {
+//   i += 1;
+// }
+//
+// // Your explanation here
+// >>>It will run on to infinty, because i = 5. Our conditional is met because 5 is greater than 3. The loop will continue running because the loop will continue to increase by 1 each time, and no limit has been given.
+//
+// 6. Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions. For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
+// for (x = 0; x < 100; x += 1) {
+//   if (x % 3 === 0) {
+//     console.log(" Fizz");
+//   } else if (x % 5 === 0) {
+//   console.log(" Buzz");
+// } else {
+//   console.log(x);
+// }
+// }
+// 7. Modify your program to log "FizzBuzz", for numbers that are divisible by both 3 and 5 (still log "Fizz" or "Buzz" for numbers divisible by only one of those).
+ for (x = 0; x < 100; x += 1){
```

Comment:
When I ran the code, it will give me values twice if a number is both multiple of 3 and 5. Ex: for 30, it will give me "30 Fizz 30 FizzBuzz". It is happening because you have two if statements inside the loop. How can we solve this problem so that the program only prints out "30 FizzBuzz"? Think about that.